addons/event_sale/tests/test_event_sale.py

```diff
@@ -121,3 +121,50 @@ def test_01_ticket_price_with_pricelist_and_tax(self):
         })
         sol.product_id_change()
         self.assertEqual(so.amount_total, 660.0, "Ticket is $1000 but the event product is on a pricelist 10 -> 6. So, $600 + a 10% tax.")
+
+    def test_02_delete_associated_sale_order(self):
```

Comment:
Better avoid "number-based" test, as it does not really add much information. Why not simply ``test_sale_order_unlink`` ? Also don't forget to add a docstring explaining purpose of the test :)